examples/drag_and_drop/main.py

```diff
@@ -0,0 +1,142 @@
+"""
+Display drag from the app into the graph widget and the event bridge.
+This is similar to the hello world sample.
+"""
+
+import sys
+
+from PyQt5 import QtWidgets
+from PyQt5.QtCore import QSize
+from PyQt5.QtGui import QDrag
+from PyQt5.QtWidgets import (
+    QGridLayout,
+    QLabel,
+    QMainWindow,
+    QPushButton,
+    QSpacerItem,
+    QVBoxLayout,
+    QWidget,
+)
+
+import qmxgraph.mime
+from qmxgraph.widget import EventsBridge, QmxGraph
+
+
+def create_drag_button(text, qmx_style, parent=None):
+    button = DragButton(parent)
+    button.setText(text)
+    # button.setIcon(...)
+    button.setProperty('qmx_style', qmx_style)
+    button.setToolTip("Drag me into the graph widget")
+    return button
+
+
+class DragButton(QPushButton):
+    """
+    Start a drag even with custom data.
+    """
+
+    def mousePressEvent(self, event):
+        mime_data = qmxgraph.mime.create_qt_mime_data(
+            {
+                'vertices': [
+                    {
+                        'dx': 0,
+                        'dy': 0,
+                        'width': 120,
+                        'height': 40,
+                        'label': self.text(),
+                        'style': self.property('qmx_style'),
+                    }
+                ]
+            }
+        )
+
+        drag = QDrag(self)
+        drag.setMimeData(mime_data)
+        # w, h = self.property('component_size')
+        # # # Image displayed while dragging.
+        # drag.setPixmap(self.icon().pixmap(w, h))
+        # # Position of the image where the mouse is centered.
+        # drag.setHotSpot(QPoint(w // 2, h // 2)
```

Comment:
Commented code

Reply:
Yes.
I didn't want to complicate the example with actual icons, but at least point out they are an option.

Reply:
I thought so, just confirming.